Orange/misc/server_embedder.py

```diff
@@ -153,32 +158,80 @@ async def embedd_batch(
         EmbeddingCancelledException:
             If cancelled attribute is set to True (default=False).
         """
-        requests = []
+        # in Orange 3.33 keep content of the if - remove if clause and complete else
+        if processed_calback is None:
+            progress_items = iter(linspace(0, 1, len(data)))
+
+            def success_callback():
+                """Callback called on every successful embedding"""
+                callback(next(progress_items))
+
+        else:
+            warnings.warn(
+                "process_callback is deprecated and will be removed in version 3.33, "
+                "use callback instead",
+                FutureWarning,
+            )
+            success_callback = processed_calback
```

Comment:
This won't work due to a typo. :) If you are still supporting it (even for 1 version), please add a test.